src/scroll-timeline-css-parser.js

```diff
@@ -415,7 +415,7 @@ export class StyleParser {
 
   eatComment(p) {
     this.assertString(p, "/*");
-    this.eatUntil("*/", p);
+    this.eatUntil("*/", p, true);
```

Comment:
Does the container query polyfill have the same bug?  If so, please file an issue.

Reply:
Will take care of it.